src/libutil/util.hh

```diff
@@ -387,6 +388,24 @@ string replaceStrings(const std::string & s,
 std::string rewriteStrings(const std::string & s, const StringMap & rewrites);
 
 
+/* Create a string from a Range (like std::vector), separated by "delimited" */
+template <typename Range, typename Value = typename Range::value_type>
+std::string join(Range const& elements, const char *const delimiter) {
```

Comment:
This looks a lot like the `concatStringsSep` function in util.cc.

Reply:
Indeed! Removed